test/blackbox-tests/test-cases/inline_tests/run.t

```diff
@@ -1,9 +1,11 @@
   $ env -u OCAMLRUNPARAM dune runtest simple
            run alias simple/runtest (exit 2)
   (cd _build/default/simple && .foo_simple.inline-tests/run.exe)
-  Fatal error: exception File "simple/.foo_simple.inline-tests/run.ml-gen", line 1, characters 10-16: Assertion failed
+  Fatal error: exception File "simple/.foo_simple.inline-tests/run.ml-gen", line 1, characters 40-46: Assertion failed
   [1]
 
+  $ env -u OCAMLRUNPARAM dune runtest simple --profile release
```

Comment:
Could you comment this test? It's a bit confusing to understand when it has no output.